CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 - update `ionoscloud_user` documentation. Fix `administrator` and add other fields description
 - change to have nested lists show correctly in tf registry docs
 
+### Enhancements
+- add configurable fields to `ionoscloud_share` resource. Fields that can be set on creation: `description`, `sec_auth_protection`, `licence_type`. 
```

Comment:
@cristiGuranIonos I know that this is an annoying comment, but please use capital letter in the beginning of the sentence. We are doing this for every sentence in the `CHANGELOG.md` and I think we should to the same here.

Reply:
done for the whole file